Library/Sender.ts

```diff
@@ -128,59 +137,99 @@ class Sender {
     public saveOnCrash(payload: string) {
         this._storeToDiskSync(payload);
     }
-    
-    private _confirmDirExists(direcotry: string, callback: (err) => void): void {
-        fs.exists(direcotry, (exists) => {
+
+    /**
+     * Send all temp files from last crash.
+     * reads all files that exist, deletes them and tries to send its load
+     */
+    public sendAllFilesOnDisk(): void {
+        var tempDir = Sender.OFFLINE_DIRECTORY || path.join(os.tmpDir(), Sender.TEMPDIR);
```

Comment:
Could you check if `_enableOfflineMode` is set to `True `before sending files from disk? I've noticed that currently we only check it in the Sender.send() method. IMHO we should not store any data on disk if offline mode is disabled.